src/components/configure/keymap/Keymap.tsx

```diff
@@ -282,6 +295,46 @@ function Layer(props: LayerProps) {
   );
 }
 
+const LabelLangMenuItems: {
+  labelLang: KeyboardLabelLang;
+  menuLabel: string;
+}[] = [
+  {
+    labelLang: 'us',
+    menuLabel: 'US',
+  },
+  { labelLang: 'jp', menuLabel: 'JIS' },
+];
```

Comment:
@adamrocker I think that this `LabelLangMenuItems` should be defined as an individual ts file, because we can find the extension point easily when we want to add other languages. Is it possible?

Reply:
Sure! I'm gonna change the data structure in order to extends lang settings easily.

Reply:
@yoichiro I put some label-language codes into below class.
3fbc884#diff-0b5a8e344e5c38d8f659bdf5b8d6959e4402dceb095e35764d6dd8fd3cc37b4b

All you need to add an item to KEY_LABEL_LANGS array if you wanna add another language.

Reply:
@adamrocker Thank you for this modification! LGTM.